docs/client-options.md

```diff
@@ -118,6 +118,7 @@ third party clients.
 - [Certify](https://github.com/webprofusion/Certify) GUI (.NET, WinForms)
 - [oocx/acme.net](https://github.com/oocx/acme.net) (.NET)
 - [kelunik/acme-client](https://github.com/kelunik/acme-client) (PHP)
+- ZeroSSL: [Portable Perl version](https://zerossl.com/installation.html#portablePerl), [Portable Win32/Win64 binaries](https://github.com/do-know/Crypt-LE/releases).
```

Comment:
Well, the reason behind mentioning Perl there is that it points to specific instructions in regards to using it as is on Windows with portable Perl - it might not be quite clear for someone that this will work on Windows otherwise. It is not a problem to remove that of course, however I can see that there are other clients listed in both places in a similar manner - for example https://github.com/kelunik/acme-client :)